### PR TITLE
Update direct-routing-plan.md

### PR DESCRIPTION
**Pull request by carolynblanding**
@NMuravlyannikov 

We're getting cases where customers believe they don't need SfB Plan 2 and are removing from the license assigned to users. Can we update the licensing information here to reflect that if included SfB Plan 2 must be assigned?

### CODE REVIEW

**Comment by NMuravlyannikov** [comment]
I'll review this was before the change we made. I need to sync with Roy and BJ on this

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/direct-routing-plan.md [hunk 1]
```diff
@@ -81,9 +81,12 @@ Firewall IP addresses and ports for Microsoft Teams media |For more information,
 Users of Direct Routing must have the following licenses assigned in Office 365: 
 
 - Microsoft Phone System 
-- Microsoft Teams 
+- Microsoft Teams + Skype for Business Plan 2 if included in Licensing Sku
```
Comment (nenonix) [inline Teams/direct-routing-plan.md]:
```suggestion
- Microsoft Teams + Skype for Business Plan 2 if included in Licensing SKU
```

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**